packages/connectivity/src/scp-cf/destination/destination-service.ts

```diff
@@ -212,6 +212,62 @@ export async function fetchCertificate(
   }
 }
 
+/**
+ * Fetch a single destination by name without credentials.
+ * @internal
+ */
+export async function fetchDestinationByNameWithoutTokens(
+  destinationServiceUri: string,
+  serviceToken: string,
+  options: DestinationServiceOptions
+): Promise<Omit<Destination, 'AuthAndExchangeTokens'> | undefined> {
```

Comment:
[q] In the api reference, I coudn't find a property `AuthAndExchangeTokens` in Destinations, is it intended to be `authTokens`?

Reply:
I'm not sure if there are security implications of omitting or not omitting that field here. Another motivation might be to keep the response "clean" from an api desgin perspective. I don't know the context well enough to judge this, but if no one sees an issue with `Promise<Destination | undefined>` that's fine with me.

Reply:
> [q] So why not just `Promise<Destination | undefined>` then?

Changed it to that now after discussing with @deekshas8